.swiftlint.yml

```diff
@@ -100,8 +100,8 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - Storage/ThirdParty/SwiftData.swift
   - Sync/Generated/Metrics.swift
   - ThirdParty
-  - UITests/
-  - XCUITests/
```

Comment:
Changing path for now, I will work to fix swiftlint in the XCUITests in a follow up PR if that sounds good